### PR TITLE
NT SERVICE\WdiServiceHost was missing in Doc

### PR DESCRIPTION
**Pull request by joinimran**
The default value for this policy is Administrators and NT SERVICE\WdiServiceHost where as NT SERVICE\WdiServiceHost was missing in the doc. I have updated the doc accordingly.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. This is based on issue #3155? Thanks.

File: windows/security/threat-protection/security-policy-settings/profile-system-performance.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Computer Configuration\\Windows Settings\\Security Settings\\Local Policies\\Use
 
 ### Default values
 
-By default this setting is Administrators on domain controllers and on stand-alone servers.
+By default this setting is Administrators and NT SERVICE\WdiServiceHost on domain controllers and on stand-alone servers.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/security-policy-settings/profile-system-performance.md]: By **default,** this setting is Administrators and NT SERVICE\WdiServiceHost on domain controllers and on stand-alone servers.

Kindly insert comma after the word "default".

**Comment by mypil** [comment]
@Justinha - can you please assist to check, approve and merge the content for this PR based on issue #3155?

Thank you.

**Comment by Justinha** [comment]
@joinimran Thanks!

**Pull request merged**